lib/src/browser.dart

```diff
@@ -89,11 +89,15 @@ class _Chrome extends Browser {
   static Version _getVersion(NavigatorProvider navigator) {
     Match match = new RegExp(r"Chrome/(\d+)\.(\d+)\.(\d+)\.(\d+)\s")
         .firstMatch(navigator.appVersion);
```

Comment:
It might still be useful to account for the scenario where the version cannot be parsed, but I think we should also do this